src/app.py

```diff
@@ -63,58 +62,60 @@ def plot_gender_chart(q_selection="mental_health_benefits_employer"):
 )
 def plot_work_interfere_bars(age_slider=[15, 65], gender="all"):
     plot_data = data
-    # To filter for responses that indicated they had a mental health condition:
+    # To apply filters to the plot data:
     plot_data = plot_data.query(
-        'work_interfere_treated != "Not applicable to me" & work_interfere_not_treated != "Not applicable to me"'
+        'work_interfere_treated != "Not applicable to me" & work_interfere_not_treated != "Not applicable to me" & age >= @age_slider[0] & age <= @age_slider[1]'
```

Comment:
This is minor but I think refactoring the last portion of conditions that you added in this PR into something like below would look cleaner:
`& @age_slider[0]  <= age <= @age_slider[1] `

Reply:
Just added this change